tests/core/mempool/test_mempool.py

```diff
@@ -2722,3 +2723,57 @@ def test_spends_by_feerate(items: List[MempoolItem], expected: List[Coin]) -> No
         assert mi.spend_bundle.coin_spends[0].coin == expected_coin
         assert last_fpc is None or last_fpc >= mi.fee_per_cost
         last_fpc = mi.fee_per_cost
+
+
+def rand_hash() -> bytes32:
+    rng = random.Random()
+    ret = bytearray(32)
+    for i in range(32):
+        ret[i] = rng.getrandbits(8)
+    return bytes32(ret)
+
+
+def item_cost(cost: int, fee_rate: float) -> MempoolItem:
+    fee = cost * fee_rate
+    amount = int(fee + 100)
+    coin = Coin(rand_hash(), rand_hash(), amount)
```

Comment:
why not use `secrets.token_bytes(32)`?

Reply:
I copy pasted this function from a place where I wanted deterministic values. I suspect this is a little bit faster too.